src/zarr/meta.py

```diff
@@ -459,7 +459,7 @@ def _encode_storage_transformer_metadata(
 
     @classmethod
     def _decode_storage_transformer_metadata(cls, meta: Mapping) -> "StorageTransformer":
-        from zarr.tests.test_storage_v3 import DummyStorageTransfomer
+        from tests.test_storage_v3 import DummyStorageTransfomer
```

Comment:
Why is a class from tests imported in user-facing functionality?

Reply:
we don't know! see https://github.com/zarr-developers/zarr-python/pull/1601

Reply:
Oh, I see, haha! No worries, I'll rebase this once your PR is merged!